README.md

```diff
@@ -1,13 +1,13 @@
-Servo-experiments
-
-A collection of experiments for Servo.  [Get Servo Nightly](https://servo-builds.s3.amazonaws.com/index.html).
+# Servo-experiments: A collection of experiments for Servo.
 
 These demos aim to show off Servo's capabilities.
 
 ## Running Servo
 * The easiest way to run Servo is to get [Servo nightly](http://download.servo.org).
-* Alternately, clone source from the [Servo repos.](https://github.com/servo/servo) and follow instructions there.
+* Alternately, follow the [guide](https://github.com/servo/servo#readme) to run servo locally
+
 
+## Running experiment locally
```

Comment:
I guess we should explain how to run them if you get it from the nightly build, as we mention it's the easiest way, we have to explain that people have to run `./servo`.